### PR TITLE
AddIsLiftable and AddLift for SkeletalFinSets

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

**Comment by mohamed-barakat** [comment]
I don't know why the code coverage dropped. As far as I can see all the lines of the new code are used in the tests I provided.

File: gap/SkeletalFinSetsForCAP.gi [hunk 1]
```diff
@@ -264,6 +264,35 @@ AddIsMonomorphism( SkeletalFinSets,
...
+    
+end );
+
+##
+AddLift( SkeletalFinSets,
+  function ( f, g )
+    local S, T;
+    
+    if not IsLiftable( f, g ) then
+        return fail;
```
Comment (zickgraf) [inline gap/SkeletalFinSetsForCAP.gi]: This line is not covered by the tests. Could you add a test?

**Comment by zickgraf** [comment]
Note: you can find lines not covered by tests by clicking on the filename in the Codecov Report (e.g. https://codecov.io/gh/homalg-project/FinSetsForCAP/pull/34/diff?src=pr&el=tree#diff-Z2FwL1NrZWxldGFsRmluU2V0c0ZvckNBUC5naQ==) and looking for lines which have a red bar on the left.

**Comment by mohamed-barakat** [comment]
Thanks, done.

**Comment by zickgraf** [comment]
Thanks :-)

**Pull request merged**